### ISSUE TITLE
Docker docs incorrect?

### ISSUE THREAD

**Issue by bard** [bats-core/bats-core#437]
The docs at https://github.com/bats-core/bats-core/blob/master/docs/source/docker-usage.md and https://bats-core.readthedocs.io/en/latest/docker-usage.html say:

```
To mount your tests into the container, first build the image as above. Then, for example with bats:

$ docker run -it -v "$PWD:/opt/bats" bats/bats:latest /opt/bats/test
```

When I try that, I get the same error from #210:

```
bash: /usr/sbin/bats: No such file or directory
```

#210 was closed saying that this was a doc issue addressed by #274, which said:

```
To run a test suite from a directory called `test` in the current directory of
your local machine, mount in a volume and direct Bats to its path inside the
container:

    $ docker run -it -v "${PWD}:/code" bats/bats:latest test
```

That didn't work for me, either, and appears not to be in the current docs.

What worked for me was passing the full in-container path to the test directory:

```
    $ docker run -it -v "$PWD:/code" bats/bats:latest /code/test
```

My directory layout:

```
~/project$ tree
├── src
│   └── foo.bash
├── test
│   └── test.bats
```

**Comment by martin-schulze-vireso** [comment]
Thanks for pointing that out. The docs are obviously inconsistent. It seems the "fix" mentioned in #210 was more or less inconsistent too. It seems these lines were done using bats itself which does not create a problem when mounting it over itself in the container.

I think the way you describe it is the only correct one of the three mentioned. You are welcome to create a PR for this. I don't know when I'll get around to.

**Comment by bard** [comment]
Done, although I stopped using the docker image since it's based on busybox so most of the code I was writing wouldn't run.
